backend/src/main/resources/db/changelog/v-1.1/140-drop-created_updated-columns.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE `keel_types`
```

Comment:
Maybe better was create only one changeset and add all queries to one sql file ?